src/test/java/org/opensearch/security/ssl/SecuritySSLReloadCertsActionTests.java

```diff
@@ -63,98 +69,70 @@ public class SecuritySSLReloadCertsActionTests extends SingleClusterTest {
         )
     );
 
-    @Test
-    public void testReloadTransportSSLCertsPass() throws Exception {
-        final String pemCertFilePath = testFolder.newFile("node-temp-cert.pem").getAbsolutePath();
-        final String pemKeyFilePath = testFolder.newFile("node-temp-key.pem").getAbsolutePath();
-        FileHelper.copyFileContents(FileHelper.getAbsoluteFilePathFromClassPath("ssl/reload/node.crt.pem").toString(), pemCertFilePath);
-        FileHelper.copyFileContents(FileHelper.getAbsoluteFilePathFromClassPath("ssl/reload/node.key.pem").toString(), pemKeyFilePath);
+    private String pemCertFilePath;
+    private String pemKeyFilePath;
+    private final String defaultCertFilePath = "ssl/reload/node.crt.pem";
+    private final String defaultKeyFilePath = "ssl/reload/node.key.pem";
+    private final String newCertFilePath =  "ssl/reload/node-new.crt.pem";
+    private final String newKeyFilePath = "ssl/reload/node-new.key.pem";
+
+    @Before
+    public void setUp() throws IOException {
+        pemCertFilePath = testFolder.newFile("node-temp-cert.pem").getAbsolutePath();
+        pemKeyFilePath = testFolder.newFile("node-temp-key.pem").getAbsolutePath();
+    }
 
-        initTestCluster(pemCertFilePath, pemKeyFilePath, pemCertFilePath, pemKeyFilePath, true);
+    @After
+    public void checkClusterHealth() throws Exception {
+        RestHelper rh = getRestHelperAdminUser();
 
-        RestHelper rh = restHelper();
-        rh.enableHTTPClientSSL = true;
-        rh.trustHTTPServerCertificate = true;
-        rh.sendAdminCertificate = true;
-        rh.keystore = "ssl/reload/kirk-keystore.jks";
+        String clusterHealthResponse = rh.executeSimpleRequest("_cluster/health");
+        JSONParser parser = new JSONParser();
+        JSONObject clusterHealthResponseJson = (JSONObject) parser.parse(clusterHealthResponse);
+        Assert.assertEquals("green", clusterHealthResponseJson.get("status"));
+
+        String catNodesResponse = rh.executeSimpleRequest("_cat/nodes?format=json");
+        JSONArray catNodesResponseJson = (JSONArray) parser.parse(catNodesResponse);
+        Assert.assertEquals(3, catNodesResponseJson.size());
```

Comment:
Rather than hardcode this number, can we pull it from the SingleClusterTest's properties?

Reply:
Thanks, updated.